components/FullEventCard/FullEventCard.js

```diff
@@ -19,6 +19,17 @@ export function FullEventCard(props) {
     }
   }
 
+  const convertDateToPrintMonth = data => {
+    let dateKey = {
+      '01': 'Jan', '02': 'Feb', '03': 'Mar',
+      '04': 'Apr', '05': 'May', '06': 'Jun',
+      '07': 'Jul', '08': 'Aug', '09': 'Sep',
+      '10': 'Oct', '11': 'Nov', '12': 'Dec'
+    }
+  let array = data.split('-')
+  return `${dateKey[array[1]]} ${array[2]} ${array[0]}`
+  }
+
```

Comment:
Cool! We got this function implementing and working. Maybe we could also find a way to add the current day of the week to the display?